test/magpiego/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/radius-project/radius/test/magpiego
 
-go 1.22
+go 1.22.0
```

Comment:
I changed this because I was getting an error trying to build this image on arm64

Reply:
Interesting. Can you share the error message? It was working fine for me.

Reply:
go: downloading go1.22 (darwin/arm64)
go: download go1.22 for darwin/arm64: toolchain not available